src/my_list.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'my_enumerable'
+
+class MyList
+  @list = []
+  include MyEnumerable
+
+  def each(&block)
+    @list.each(&block)
+  end
+
+  def initialize(*lis)
+    @list = lis
+  end
+end
+
+# Create our list
+list = MyList.new(1, 2, 3, 4)
+#=> #<MyList: @list=[1, 2, 3, 4]>
+
+# Test #all?
+list.all? { |e| e < 5 }
+# => true
+list.all? { |e| e > 5 }
+# => false
+
+# Test #any?
+list.any? { |e| e == 2 }
+# => true
+
+list.any? { |e| e == 5 }
+# => false
+
+# Test #filter
+list.filter(&:even)
+# => [2, 4]

```

Comment:
- [ ] Let us fix the following issues by adding the `?` sign after the filter and even methods.

![image](https://user-images.githubusercontent.com/55635977/162447828-f46b4723-36f3-4269-8ed2-0ef8f7bd14ee.png)

![image](https://user-images.githubusercontent.com/55635977/162447975-2d9e6902-c57a-4f7f-8f56-b8d804e1e44c.png)